docs/detections/alert-suppression.asciidoc

```diff
@@ -9,17 +9,31 @@ Alert suppression requires a https://www.elastic.co/pricing[Platinum or higher s
 preview::[]
 --
 
-Alert suppression allows you to reduce the number of repeated or duplicate detection alerts created by a <<create-custom-rule,custom query detection rule>> each time the rule runs. 
+Alert suppression allows you to reduce the number of repeated or duplicate detection alerts created by a <<create-custom-rule,custom query detection rule>>.
```

Comment:
```suggestion
Alert suppression allows you to reduce the number of repeated or duplicate detection alerts created by a <<create-custom-rule,custom query>> detection rule. 
```
Feel free to ignore, but I was thinking of our conversation yesterday and I wonder if having the "custom query" part set apart from the "detection rule" part would be a nice subtle reminder that "custom query" is the name for a rule type